returnn/training.py

```diff
@@ -189,7 +211,7 @@ def _get_run_cmd(self):
             self.out_returnn_config_file.get_path(),
         ]
 
-        if self.use_horovod:
+        if self.horovod_num_processes:
             run_cmd = [
```

Comment:
Does the `run_cmd` not need information from the `os.environ["PE_HOSTFILE"]` to decide to which nodes to connect?

I remember that when running multi-gpu multi-node training manually, I provided the option `-H localhost,<other_host_ip>`. 

Ah, I see there is also the option `--hostfile`.

Reply:
Ok, thanks for clarification. Yes, I was running manually on just two nodes outside of a cluster (and outside of sisyphus)

Reply:
I added a comment.